pages/test-evaluate/evaluation-tools/selecting-eval-tools.pl.md

```diff
@@ -0,0 +1,230 @@
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Treść strony" class="full" %}
+{:/}
+
+- TOC jest tworzony automatycznie.
+{:toc}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Wideo: Omówienie narzędzi oceny {#video}
+
+{% include video-player-data.html
+    yt-id="bn1XJSjc_qM"
+    video-id="tools-for-evaluating"
+%}
+
+_Ten film jest również dostępny na serwerze W3C: [Wideo: Omówienie narzędzi oceny (format pliku: MP4, rozmiar pliku: 51MB)](https://media.w3.org/wai/evaluation-intros/tools-for-evaluating.mp4)._
+
+
+{% include excol.html type="start" id="video-intro-transcript" %}
+
+Transkrypcja oraz opis wizualnej strony filmu
+
+{% include excol.html type="middle" %}
+
+<table>
+  <thead>
+    <tr>
+      <th width="50%">Dźwięk</th>
+      <th>Obraz</th>
+    </tr>
+  </thead>
+<tbody>
+  <tr>
+    <td>Narzędzia oceny dostępności cyfrowej</td>
+    <td>Narzędzia oceny dostępności cyfrowej</td>
+  </tr>
+  <tr>
+    <td>Istnieją programy i usługi internetowe, które pomogą Ci rozpoznać problemy z dostępnością.</td>
+    <td>Otwiera się skrzynka narzędziowa przy komputerze. Ze skrzynki wychodzi lupa, aby sprawdzić stronę internetową na komputerze. Lupa powiększa słowo „dostępność”.</td>
+  </tr>
+  <tr>
+    <td>Mogą one zaoszczędzić wiele czasu i wysiłku podczas oceny oraz zapobiec powstawaniu nowych barier dostępności.</td>
+    <td>Oszczędzaj czas i wysiłek. Unikaj barier dostępności.</td>
+  </tr>
+  <tr>
+    <td>Jednak narzędzia nie mogą zrobić wszystkiego. Niektórych testów dostępności po prostu nie można zautomatyzować i muszą być przeprowadzone ręcznie.</td>
+    <td>Osoba obok komputera ze stroną internetową używa lupy ze słowem „dostępność”.</td>
+  </tr>
+  <tr>
+    <td>Niektóre narzędzia poprowadzą Cię przez te testy, których nie można zautomatyzować.</td>
+    <td>Lista kontrolna obok komputera.</td>
+  </tr>
+  <tr>
+    <td>Niektóre narzędzia sprawdzają tylko jedną stronę na raz, inne mogą sprawdzać całe witryny.</td>
+    <td>Strona w witrynie internetowej jest skanowana. Nad elementami strony pojawia się kolejno znak błędu, powodzenia i znak zapytania.</td>
+  </tr>
+  <tr>
+    <td>Narzędzia można integrować z różnymi środowiskami pracy. Na przykład z przeglądarką internetową, systemem zarządzania treścią (C-M-S) oraz w narzędziach programistycznych i wdrożeniowych.</td>
+    <td>Słowa „Przeglądarka internetowa”, „CMS” i „środowisko wdrożeniowe” wokół ikony narzędzi wyświetlanej na komputerze.</td>
+  </tr>
+  <tr>
+    <td>Wspierają różne role w zespole projektowym, takie jak autorów treści, twórców kodu, projektantów oraz właścicieli produktów.</td>
+    <td>Ikona narzędzi jest otoczona czterema ikonami: długopisu, kodu, pędzla i osoby z kluczem.</td>
+  </tr>
+  <tr>
+    <td>Trzeba pamiętać, że w niektórych przypadkach narzędzia mogą dawać nieprawidłowe wyniki.</td>
+    <td>Lupa powiększająca znak ostrzegawczy z wykrzyknikiem. Wyświetlanych jest wiele lup. </td>
```

Comment:
@zwiastunsw This description seems to have been reworded. Would you think of an alternative to get closer to the original wording?